### PR TITLE
[styleguide] add transform imports plugin

### PR DESCRIPTION
**Pull request by simonz-git**

### CODE REVIEW

File: styleguide/babel.config.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
 module.exports = function (api) {
   const config = getConfig(api, {
     alias: {
       ui: '@startupjs/ui'
     }
   })
+
+  config.plugins = [
+    ...config.plugins,
+    ['@gzaripov/babel-plugin-transform-imports', {
```
Comment (pturchik) [inline styleguide/babel.config.js:15]: '@gzaripov/transform-imports'

File: styleguide/babel.config.js [hunk 2]
```diff
@@ -1,10 +1,28 @@
 const getConfig = require('startupjs/bundler').babelConfig
 
+const noImportConversion = (importName, isBool) => {
+  const headers = new Set(['H1', 'H2', 'H3', 'H4', 'H5', 'H6'])
+  if (headers.has(importName)) return isBool ? true : '/Headers'
+  return isBool ? false : `/${importName}`
+}
+
```
Comment (pturchik) [inline styleguide/babel.config.js:5]:
```js
const HEADERS_LIST = ['H1', 'H2', 'H3', 'H4', 'H5', 'H6']
const isHeader = (name) => HEADERS_LIST.includes(name)
```

File: styleguide/babel.config.js [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
       ui: '@startupjs/ui'
     }
   })
+
+  config.plugins = [
+    ...config.plugins,
+    ['@gzaripov/babel-plugin-transform-imports', {
+      '@startupjs/ui': {
+        transform: importName => `@startupjs/ui/components${noImportConversion(importName)}`,
+        skipDefaultConversion: importName => noImportConversion(importName, true),
```
Comment (pturchik) [inline styleguide/babel.config.js]:
```js
transform: importName => `@startupjs/ui/components` + (isHeader(importName) ? path + '/Headers' : '')
skipDefaultConversation: isHeader
```

**Review by pturchik — CHANGES_REQUESTED** [review]

File: styleguide/babel.config.js (hunk 1 above)
Comment (simonz-git) [reply styleguide/babel.config.js:15]: done

File: styleguide/babel.config.js (hunk 2 above)
Comment (simonz-git) [reply styleguide/babel.config.js:5]: done

File: styleguide/babel.config.js (hunk 3 above)
Comment (simonz-git) [reply styleguide/babel.config.js]: done

**Review by pturchik — APPROVED** [review]

**Pull request merged**